### ISSUE TITLE
Wrong installation pathes for breeze and breeze-dark icons

### ISSUE THREAD

**Issue by rapgro** [nitroshare/nitroshare-desktop#80]
The path "breeze-dark/apps/22/nitroshare-indicator.svg" looks wrong to me.
Better use "breeze-dark/scalable/apps/nitroshare-indicator.svg"
Analogously for breeze instead of breeze-dark.

**Comment by rapgro** [comment]
Fix with this script:
`mkdir -p src/dist/icons/breeze-dark/scalable src/dist/icons/breeze/scalable
mv src/dist/icons/breeze-dark/apps/22 src/dist/icons/breeze-dark/scalable/apps
mv src/dist/icons/breeze/apps/22 src/dist/icons/breeze/scalable/apps
rm -rfv src/dist/icons/breeze\*/apps`

**Comment by nathan-osman** [comment]
No, this is correct. The "breeze" icons are shown on a light background and must use a dark color to stand out from the background. Conversely, the "breeze-dark" icons are shown on a dark background and must show a light color to stand out.

**Comment by rapgro** [comment]
I think you missed the point. It's not about removing the icons, it's about a wrong directory name they get installed to. Your directory structure is against XDG standard.
https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#directory_layout

**Comment by nathan-osman** [comment]
Oh, I'm sorry. I misunderstood what you were saying. I'll take a look at this tomorrow.

**Comment by rapgro** [comment]
not addressed in 0.3.3

**Comment by abitrolly** [comment]
Is that still actual? @rapgro maybe send a patch?

**Comment by rapgro** [comment]
not addressed in 0.3.4
https://src.fedoraproject.org/rpms/nitroshare/blob/master/f/nitroshare.spec#_107

**Comment by abitrolly** [comment]
@rapgro send it here - https://github.com/nitroshare/nitroshare-desktop/pulls
